### ISSUE TITLE
Add alt tags to images in Sanity

### ISSUE THREAD

**Issue by sarahfossheim** [oslopride/oslopride.no#133]
For accessibility reasons, all images should have alt tags. Suggest we add alt text as a required field for images in Sanity.
Eg:
```JS
export default {
    title: 'Image',
    name: 'mainImage',
    type: 'image',
    fields: [
      {
        title: 'Alternative Text',
        name: 'alt',
        type: 'string',
        description: 'Important for SEO and accessibility.',
        validation: Rule => Rule.error('You have to fill out the alternative text.').required()
      }
    ]
  }
```

**Comment by sarahfossheim** [comment]
Didn't notice there was already an issue for this: https://github.com/oslopride/oslopride.no/issues/103
